components/binary_sensor/mpr121.rst

```diff
@@ -32,6 +32,7 @@ required to be set up in your configuration for this sensor to work.
 
     binary_sensor:
       - platform: mpr121
+        mpr121_id: mpr121_component
```

Comment:
This is ok however we don't clutter the docs first example so better this should not go here 

Btw a new check in esphome will make it fail to compile if you declare two hubs and don't specify the one you want to use. So future people will have a better time than you. 

The definition below is ok